path-animate/demo.html

```diff
@@ -97,19 +97,32 @@
         stage.update();
     }
 
+    function distanceBetweenPoints(xDist, yDist) {
+
+        dist = Math.sqrt(Math.pow(xDist, 2) + Math.pow(yDist, 2));
+
+        return dist;
+    }
+
+    function angleOfObject(xDist, yDist) {
+        angle = Math.atan(yDist/xDist) * (180 / Math.PI);
+        (xDist < 0) ? angle += 90 : angle -= 90;
```

Comment:
add a comment here to explain why we are adjusting the angle by 90 degrees, as this seems arbitrary when used inside a function like this.

Reply:
I do not think we want the 90 degree adjustment here, probably should go outside this function?